docs/source/sdk.rst

```diff
@@ -139,7 +140,14 @@ execute
       * gasLimit : number - limit of gas to refund
     - **privateKey** : string - a private key to be used to sign the transaction and has permission to execute message
   Returns:
-    `promise`, that resolves to the hash of the on-chain transaction
+    `promise`, that resolves to the ``Execution``
+
+.. _execution:
+
+  **Execution** contains:
+
+  - **messageStatus** - current status of sent message (:ref:`learn more<messageStatus>`)
+  - **waitToBeMined** - a function that returns a promise, that resolves to MessageStatus as soon as transaction that contains this message is mined
```

Comment:
`a function that returns a promise, that resolves to MessageStatus once transaction enclosed with Message is mined`